test/e2e/windows/gmsa.go

```diff
@@ -0,0 +1,160 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package windows
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+// We need an image that packs the `nltest` command for this - and the `winPauseImageName` one doesn't.
+// I'm afraid this is the smallest one that does.
+const gMSATestImageName = "mcr.microsoft.com/powershell"
```

Comment:
yeah too bad nltest is not part of the nanoserver images...
but also nl*.dll from system32 isn't either.

Reply:
Same comment as above. No need to hardcore. We should just use one of the images in "k8s.io/kubernetes/test/utils/image", they are all based on windowsservercore , thus will have powershell. If not, we can just define another image in "k8s.io/kubernetes/test/utils/image", either way I suggest we stick to having all image names and repos there.

Reply:
Neat, hadn't seen the trick of overriding the repos. Thanks.